src/components/App.js

```diff
@@ -0,0 +1,21 @@
+import '../App.css';
+import Button from './Button';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+function App() {
+  return (
+    <>
+      <div className="App">
+        <header className="App-header">
+          <h3>REACT CALCULATOR</h3>
+        </header>
+        <Display result="5 * 5" />
+        <ButtonPanel />
+        <Button name="Calculator" />
```

Comment:
**[Required]** Button should be imported and rendered in the ButtonPanel component, not in the App component. [Reference](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/addToComponent.pngl)

**[Required]** Commit messages should be capitalized. [Reference](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)